docs/tutorials/basic/go.md

```diff
@@ -148,26 +148,16 @@ message Point {
 
 Next we need to generate the gRPC client and server interfaces from our .proto
 service definition. We do this using the protocol buffer compiler `protoc` with
-a special gRPC Go plugin.
+a special gRPC Go plugin. 
+This is similar to what we did in the [quickstart guide](https://github.com/grpc/grpc.github.io/blob/GA/docs/quickstart/go.md)
```

Comment:
Should this link point to the quickstart html page (which doesn't exist now) instead of the md file?